examples/use-cases/LocalForwarderHealing/README.md

```diff
@@ -0,0 +1,159 @@
+# Test kernel to kernel connection
```

Comment:
Please create a suite for healing tests and add this test into the suite.

Reply:
ok, done